### ISSUE TITLE
Unable to run bundled Omnisharp

### ISSUE THREAD

**Issue by csholmq** [OmniSharp/omnisharp-vscode#321]
From https://github.com/Microsoft/vscode/issues/6319#issuecomment-219382560

"Built-in" Omnisharp extension states

>Don't install this extension unless you want to try an early version. By default it is already
bundled with VS Code.

Which isn't true anymore. Something has broken in either latest Insiders build (1.1.0-insider) or Omnisharp extension. Had to install Omnisharp extension explicitly.

*Also, installing the C# extension was a pain and required several `ext install csharp`.*

**Comment by DustinCampbell** [comment]
The Built-in Omnisharp extension isn't supported or being updated any longer. The source has transitioned to the C# extension, so that's the latest.

Very sorry to hear that you had trouble installing the extension. What were the issues you encountered? Did it just not work at all? Did you happen to capture any OmniSharp logs from the output window?

**Comment by csholmq** [comment]
Not more than "Error: Failed to start OmniSharp". I will try to reproduce and get more logs later this week. 

Issue was sorted installing the Omnisharp ext. So something must be missing in the C# ext.

**Comment by DustinCampbell** [comment]
FWIW, I've renamed the "Built-in Omnisharp" extension and added text to set expectations and set it up properly [here](https://marketplace.visualstudio.com/items?itemName=ms-vscode.omnisharp).

The C# extension does not include OmniSharp. It downloads and install the correct OmniSharp for the operating system you're running on. I'm wondering if the download is failing for some reason. What OS are you running on?

**Comment by csholmq** [comment]
Låter OSX.

**Comment by DustinCampbell** [comment]
@csholmq: I just wanted to follow up and see how things are going. We've had a few updates of the C# extension in the past several weeks and there have been several improvements made to the OmniSharp download process. Is it working better for you these days?

**Comment by csholmq** [comment]
@DustinCampbell The experience is waaaay better! I uninstalled both the C# and Omnisharp legacy extension and all CLI tools I previously hacked in. I then reinstalled the C# extension and followed the instructions to;

1. Install OpenSSL using brew
2. Install .NET Core SDK using installer

I experienced no drawbacks working on OSX. I was able to create a dummy project, edit code using IntelliSense and even debug. Super!
